lib/ansible/modules/remote_management/ucs/ucs_timezone.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: ucs_timezone
+
+short_description: Configures timezone on Cisco UCS Manager
+
+version_added: "2.7"
+
+description:
+    - Configures timezone on Cisco UCS Manager.
```

Comment:
Not sure if you knew, but list items do not need to be indented. So you can do:
```yaml
description:
- Configures timezone on Cisco UCS Manager.
```
I prefer to write minimal YAML myself, but that's just a preference.

Reply:
I didn't know, made the change